components/script/dom/element.rs

```diff
@@ -2303,6 +2303,16 @@ impl VirtualMethods for Element {
         Some(self.upcast::<Node>() as &VirtualMethods)
     }
 
+    fn attribute_is_mapped(&self, attr: &Attr) -> bool {
+        // FIXME: This should be more fine-grained, not all elements care about these.
+        if attr.local_name() == &local_name!("width") ||
+            attr.local_name() == &local_name!("height") {
+                return true;
```

Comment:
nit: Indentation is off here.

Reply:
Thanks, vim indents it the wrong way for whatever reason.. fixed and pushed!